packages/react-components/react-components/src/index.ts

```diff
@@ -462,6 +462,7 @@ export {
   useMenuListContext_unstable,
   useMenuListContextValues_unstable,
   useMenuListStyles_unstable,
+  useOnMenuMouseEnter,
   useMenuPopover_unstable,
```

Comment:
```suggestion
  useMenuPopover_unstable,
```

I don't mine react-menu re-exporting this utility hook, but I don't think this utility should be exported under the suite package, since it's not necessary for our recomposition API

Reply:
Hi @ling1726 , removed utility from `react-components`